cardano-explorer/src/Explorer/Web/ClientTypes.hs

```diff
@@ -0,0 +1,264 @@
+{-# OPTIONS_GHC -fno-warn-unused-top-binds #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Do we need that?

Reply:
`    Orphan instance: instance ToString ByteString`
its encoding all `ByteStrings` to hex

Reply:
Where does `ToString` come from? Isn't there a instance for `ByteString` there?

If there isn't, adding a `BS.unpack` where needed is still probably still a better options.